src/parser/druid/restoration/modules/shadowlands/conduits/FlashOfClarity.tsx

```diff
@@ -14,30 +14,46 @@ import ItemHealingDone from 'interface/ItemHealingDone';
 
 class FlashOfClarity extends Analyzer {
 
+
   healingBoost = 0;
   healing = 0;
 
-  targetsWithClearCastingRegrowth = [];
+  targetsWithClearCastingRegrowth: number[] = [];
 
   /**
    * Flash of Clarity makes it so any regrowth casted with clearcasting is boosted by x% on hot healing and direct heal
    */
-  constructor(...args) {
-    super(...args);
-    this.active = false;
+  constructor(options: Options) {
+    super(options);
 
-    this.healingBoost = .20;//TODO Get from combat data when they EXPORT IT >:c
+    const conduitRank = this.selectedCombatant.conduitRankBySpellID(SPELLS.JADE_BOND.id);
 
-    if (!this.active) {
+    if (!conduitRank) {
+      this.active = false;
       return;
     }
 
+    this.healingBoost = this.conduitScaling(.20, conduitRank);
```

Comment:
don't you just want to this.active = hasconduit ?
also don't like magic number in that conduitScaling function call

Reply:
didn't know we had a `hasConduit` method hype